katas/content/qubit/index.md

```diff
@@ -22,6 +22,8 @@ This kata introduces you to one of the core concepts in quantum computing - the
 - Basic knowledge of complex arithmetic
 - Basic knowledge of linear algebra
```

Comment:
Same comment about knowing knowledge, that looks like it was pervasive, I'd really like to find a better formulation throughout the katas. I'd suggest "basic concepts of linear algebra/complex arithmetic", "basic gates", and "measurements"